draft-ietf-git-using-github.md

```diff
@@ -428,6 +447,242 @@ A working group that uses GitHub MAY provide either facility at the request of
+Working Group mailing list.
+
+Working Group chairs SHOULD confirm that the Working Group has consensus to
+adopt any process.  In particular, the introduction of a more tightly-controlled
+process can have the effect of privileging positions already captured in
+documents, which might disadvantage alternative viewpoints.
+
+
+### Early Design Phases
+
+During early phases of the design of a protocol, chairs MAY allow editors to
+manage all aspects of issues.  Editors are permitted to make decisions about how
+to both identify and resolve technical issues, including making any changes that
+editors feel necessary.
+
+Chairs need to explicitly decide that this sort of process is needed and
+announce the decision to the Working Group.  In many cases, documents that are
+adopted by a Working Group are already sufficiently mature that a looser process
+is not beneficial.  The primary reason to grant editors more discretionary power
+is to improve the speed with which changes can be made.  The risk is that design
+changes might not always reflect the consensus of the Working Group.
+
+Changes made by editors under this process do not completely lack oversight.
+GitHub and git provide tools for ensuring that changes are tracked and can be
+audited.  Within the usual Working Group process it is expected that
+Internet-Drafts will receive regular review. Finally, process checkpoints like
+Working Group Last Call (WGLC; Section 7.4 of {{!RFC2418}}) provides additional
+safeguards against abuse.
+
+Working Groups are advised against allowing editors this degree of flexibility
+for the entirety of a document lifecycle.  Once a document is more stable and
+mature, it is likely appropriate to move to a more tightly controlled process.
+
+
+### Managing Mature Documents
+
+As a document matures, it becomes more important to understand not just that the
+document as a whole retains the support of the working group, but that changes
+are not made without wider consultation.
+
+Chairs might choose to manage the process of deciding which issues are
+substantive.  For instance, chairs might reserve the ability to use the `design`
+label to new issues (see {{label-design}}) and to close issues marked as `design`.
+Chairs should always allow document editors to identify and address editorial
+issues as they see fit.
+
+As documents mature further, explicit confirmation of technical decisions with
+the Working Group mailing list becomes more important.
+
+Gaining Working Group consensus about the resolution of issues can be done in
+the abstract, with editors being permitted to capture the outcome of discussions
+as they see fit.
+
+More mature documents require not only consensus, but consensus about specific
+text.  All substantive changes to documents that have passed WGLC MUST be
+proposed as pull requests, discussed on the mailing list, and have chairs
+explicitly confirm consensus.  Chairs MAY institute this stricter process prior
+to WGLC.
```

Comment:
Doesn't it seem sufficient to have chairs confirm consensus and not necessarily require pull requests? (E.g., have an editor make changes directly to HEAD... or is there a rationale for PRs that I can't read between the lines to see here?). I'm just saying making that a MUST means lots of PRs that may not have to have been? Or maybe post-WGLC we definitely want PRs?